src/category_theory/monad/coequalizer.lean

```diff
@@ -7,7 +7,7 @@ Authors: Bhavik Mehta
 import category_theory.limits.shapes.reflexive
 import category_theory.limits.preserves.shapes.equalizers
 import category_theory.limits.preserves.limits
-import category_theory.monad.adjunction
+import category_theory.monad.algebra
```

Comment:
Is this necessary?

Reply:
Not strictly necessary, but it does weaken the imports (`.algebra` is imported in `.adjunction` but we don't need any adjunction things in this file). An older version of this PR needed this change to prevent cycles but it doesn't any more. This weakening caused basically all the other changes to become necessary as well. I don't mind reverting but I think this is still a positive change, so it should at least happen in a new PR if you prefer.

Reply:
Gotcha. Yes, importing `algebra` sounds good in this case.